conans/test/integration/cache/cache2_update_test.py

```diff
@@ -303,8 +307,10 @@ def test_revision_fixed_version(self):
         # |             | REV0 (1000)|            |           |            |
         # |             |            |            |           |            |
 
-        self.client.run(f"install --requires={server_rrev}@#{server_rrev.revision} --update")
+        self.client.run(f"install --requires={server_rrev}@#{server_rrev.revision} --update='foo/*'")
+        self.client.assert_listed_require({"liba/1.0.0": "Cache (Updated date) (server0)"})
```

Comment:
This does not seem right, the date should not have been updated, nor from server 0 (next assert has it on server2!)

Reply:
It would be better if these tests doesn't need to be changed at all, only adding things, not modifying. But I need to understand better, in theory ``--update`` should update timestamps if they are newer on the server.